src/map/utils/fishingcontest.cpp

```diff
@@ -0,0 +1,716 @@
+﻿/*
+===========================================================================
+
+Selbina Fishing Contest
+
+===========================================================================
+*/
+
+#include "fishingcontest.h"
+
+#include <algorithm>
+#include <cmath>
+#include <cstring>
+#include <time.h>
+
+#include "charutils.h"
+#include "common/vana_time.h"
+#include "itemutils.h"
+#include "lua/luautils.h"
+#include "zoneutils.h"
+
+namespace fishingcontest
+{
+    FishingContest                    CurrentFishingContest; // Currently active contest
+    std::vector<FishingContestEntry*> FishingContestEntries; // Currently active contest entries
```

Comment:
If these are only used in here, put them into an anonymous namespace (I think you can see this in zone_entities)